### DISCUSSION TITLE
UART 使えませんか？

### DISCUSSION THREAD

**Discussion by ahidaka** [algyan/XIAOGYAN#5] [General]
時期的に（多分開発出来る様になってると思うので）、「Window 11 IoT Widget を作ってみよう」セミナーの教材として使えればと思うので、UARTが使えないかなと思っています。

PC<->USB-UART変換器<->algyan_8th基板

**Comment by matsujirushi** [comment]
おぉー！

（いま使おうと思っている）Seeed Studio XIAO ESP32C3はUSB-CDCとJTAGを内蔵しているので、VCOMで通信できます！
なので、USB-UART変換器不要―

**Reply by ahidaka** [reply]
素晴らしい！ありがとうございます。
